src/openforms/prefill/fields.py

```diff
@@ -35,7 +35,7 @@ def formfield(self, **kwargs):
     def _get_plugin_choices(self):
         choices = []
         for plugin in register.iter_enabled_plugins():
-            if plugin.requires_auth != self.auth_attribute:
+            if plugin.requires_auth and self.auth_attribute not in plugin.requires_auth:
                 continue
```

Comment:
```suggestion
            if self.auth_attribute not in plugin.requires_auth:
                continue
```
simplified - if the collection is empty, the check has the same outcome